runtime-sdk/src/modules/core/mod.rs

```diff
@@ -30,10 +40,206 @@ pub enum Error {
     #[error("insufficient balance to pay fees")]
     #[sdk_error(code = 5)]
     InsufficientFeeBalance,
+
+    #[error("invalid argument")]
+    #[sdk_error(code = 6)]
+    InvalidArgument,
+
+    #[error("gas overflow")]
+    #[sdk_error(code = 7)]
+    GasOverflow,
+
+    #[error("out of gas")]
+    #[sdk_error(code = 8)]
+    OutOfGas,
+
+    #[error("batch gas overflow")]
+    #[sdk_error(code = 9)]
+    BatchGasOverflow,
+
+    #[error("batch out of gas")]
+    #[sdk_error(code = 10)]
+    BatchOutOfGas,
+}
+
+/// Parameters for the accounts module.
+#[derive(Clone, Debug, Serialize, Deserialize)]
+#[serde(deny_unknown_fields)]
+pub struct Parameters {
+    #[serde(rename = "batch_gas")]
+    pub batch_gas: u64,
```

Comment:
Should this be `max_batch_gas`?

Reply:
I'm fine with max_batch_gas. that would have been my second choice after batch_gas (the latter for consistency with `gas` in fee)